airflow/contrib/hooks/gcp_bigtable_hook.py

```diff
@@ -0,0 +1,232 @@
+        :param cluster_nodes: (optional) Number of nodes for cluster.
+        :type cluster_storage_type: enums.StorageType
+        :param cluster_storage_type: (optional) The type of storage.
+        :type timeout: int
+        :param timeout: (optional) timeout (in seconds) for instance creation.
+                        If None is not specified, Operator will wait indefinitely.
+        """
+        cluster_storage_type = enums.StorageType(cluster_storage_type)
+        instance_type = enums.Instance.Type(instance_type)
+
+        instance = Instance(
+            instance_id,
+            self.get_client(project_id),
+            instance_display_name,
+            instance_type,
+            instance_labels,
+        )
+
+        clusters = [
+            instance.cluster(
+                main_cluster_id,
+                main_cluster_zone,
+                cluster_nodes,
+                cluster_storage_type
+            )
+        ]
+        if replica_cluster_id and replica_cluster_zone:
+            clusters.append(instance.cluster(
+                replica_cluster_id,
+                replica_cluster_zone,
+                cluster_nodes,
+                cluster_storage_type
+            ))
+        operation = instance.create(
+            clusters=clusters
+        )
+        operation.result(timeout)
+        return instance
+
+    # noinspection PyMethodMayBeStatic
+    def create_table(self, instance, table_id, initial_split_keys, column_families):
+        """
+        Creates the specified Cloud Bigtable table.
+        Raises google.api_core.exceptions.AlreadyExists if the table exists.
+
+        :type instance: Instance
+        :param instance: The Cloud Bigtable instance that owns the table.
+        :type table_id: str
+        :param table_id: The ID of the table to create in Cloud Bigtable.
+        :type initial_split_keys: list
+        :param initial_split_keys: (Optional) A list of row keys in bytes to use to initially split the table.
+        :type column_families: dict
+        :param column_families: (Optional) A map of columns to create. The key is the column_id str, and the
+        value is a GarbageCollectionRule.
+        """
+        table = Table(table_id, instance)
+        table.create(initial_split_keys, column_families)
+
+    def delete_table(self, project_id, instance_id, table_id):
+        """
+        Deletes the specified table in Cloud Bigtable.
+        Raises google.api_core.exceptions.NotFound if the table does not exist.
+
+        :type project_id: str
+        :param project_id: The ID of the GCP project.
+        :type instance_id: str
+        :param instance_id: The ID of the Cloud Bigtable instance.
+        :type table_id: str
+        :param table_id: The ID of the table in Cloud Bigtable.
+        """
+        instance = Instance(instance_id, self.get_client(project_id))
+        table = Table(table_id, instance)
+        table.delete()
+
+    # noinspection PyMethodMayBeStatic
```

Comment:
Can we remove this as well, if the method is static, please make it static

Reply:
Sure. Makes perfect sense.